app/controllers/api/subcollections/policies.rb

```diff
@@ -8,11 +8,19 @@ def policies_query_resource(object)
       end
 
       def policies_assign_resource(object, _type, id = nil, data = nil)
-        policy_assign_action(object, :policies, id, data)
+        if object.kind_of?(collection_class(:policy_profiles))
+          object.add_member(policy_specified(id, data, :policies, MiqPolicy))
```

Comment:
would prefer having add_policy and remove_policy provided in the MiqPolicySet model.